docs/Setup.rst

```diff
@@ -30,37 +33,34 @@ Operating Sytem and Package setup of the Deployer Node
 - Deployer OS Requirements:
     - Ubuntu (Software installation is not yet supported under Ubuntu)
         - Release 14.04LTS or 16.04LTS
-        - SSH login enabled
+        - SSH login enabled if the deployer node needs to be accessed remotely
```

Comment:
There are other ways a user could remote access a server so I don't think SSH (server) is _really_ a deployer requirement (/nickpick).

Reply:
I'm going to remove this line altogether. We have to assume some level of competence on the part of the user.